src/platformdirs/android.py

```diff
@@ -18,7 +18,7 @@ class Android(PlatformDirsABC):
     @property
     def user_data_dir(self) -> str:
         """:return: data directory tied to the user, e.g. ``/data/user/<userid>/<packagename>/files/<AppName>``"""
-        return self._append_app_name_and_version(_android_folder(), "files")
+        return self._append_app_name_and_version(_android_folder() or "", "files")
```

Comment:
Do we need the or here? I've understood that when this is called it's guaranteed that it will be not None, not?

Reply:
`tox -e type` swears

Reply:
`mypy` knows less then we, but probably a `typing.cast` for mypy likely is better choice than `or` 👍

Reply:
Yep, i think so too